File: ruby/soap_client.rb
require 'soap/wsdlDriver'

wsdl = 'https://www.dataaccess.com/webservicesserver/NumberConversion.wso?WSDL'
client = SOAP::WSDLDriverFactory.new(wsdl).create_rpc_driver

n = ARGV[0] || '10'
response = client.NumberToWords('ubiNum' => n.to_i)
puts response.NumberToWordsResult

File: node/soap_client.js
import soap from 'soap';

const wsdl = 'https://www.dataaccess.com/webservicesserver/NumberConversion.wso?WSDL';
const n = process.argv[2] || '10';

soap.createClient(wsdl, (err, client) => {
  if (err) throw err;
  client.NumberToWords({ ubiNum: parseInt(n, 10) }, (err, result) => {
    if (err) throw err;
    console.log(result.NumberToWordsResult);
  });
});
